src/.nuget/packages.config

```diff
@@ -1,5 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <packages>
+  <package id="Microsoft.DotNet.BuildTools" version="1.0.25-prerelease-00199" />
+  <package id="Microsoft.Net.Compilers" version="1.0.0" targetFramework="net451" userInstalled="true" />
```

Comment:
packages.config probably needs to go away. I remember replacing it with project.json